src/actions/deviceActions.ts

```diff
@@ -302,6 +302,24 @@ export const open =
                 dispatch(setSavePending(true));
             }
 
+            const shouldCheckDiskFull =
+                performance.now() - lastDiskFullCheck > 10_000;
+
+            if (shouldCheckDiskFull) {
+                lastDiskFullCheck = performance.now();
+                isDiskFull(
+                    getDiskFullTrigger(getState()),
+                    getSessionRootFolder(getState())
+                ).then(isFull => {
+                    if (isFull) {
+                        logger.warn(
+                            'Session stopped. Disk full trigger detected'
```

Comment:
It leaves me wondering "so what it was detected?". I would rather suggest:
``Session stopped because of the disk full trigger.``. 
Or, if you want two sentences:
``Session stopped. Disk full trigger value reached.``